h/config.py

```diff
@@ -116,6 +116,9 @@ def configure(environ=None, settings=None):  # pylint: disable=too-many-statemen
 
     settings_manager.set("h.websocket_url", "WEBSOCKET_URL")
 
+    # Reporting settings
+    settings_manager.set("h.report.fdw_users", "REPORT_FDW_USERS", type_=aslist)
```

Comment:
I'm wondering if we want to put "FDW" in all the names here? Should we be more "what" than "how" centric? These are "REPORT_USERS", which we then grant FDW access to, rather than FDW users?